testing/load/load-test.js

```diff
@@ -0,0 +1,68 @@
+
+import { check } from "k6";
+import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.2/index.js";
+import { Http, Tracetest } from "k6/x/tracetest";
+import { sleep } from "k6";
+
+export const options = {
+  stages: [
+    { duration: "5m", target: 5 },
+    { duration: "5m", target: 10 },
+    { duration: "5m", target: 5 },
+  ],
+  thresholds: {
+    http_req_duration: ["p(95)<500"],
+  },
+};
+
+const tracetest = Tracetest({
+  serverUrl: "http://localhost:11633",
+});
+const testId = "kc_MgKoVR";
+let pokemonId = 6; // charizad
```

Comment:
nit: Just because my OCD: 

```suggestion
let pokemonId = 6; // charizard
```